code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -199,30 +200,29 @@
 	///Bool for whether this mech can only be used on lavaland
 	var/lavaland_only = FALSE
 
-	/// Ui size, so you can make the UI bigger if you let it load a lot of stuff
-	var/ui_x = 1100
-	/// Ui size, so you can make the UI bigger if you let it load a lot of stuff
-	var/ui_y = 600
 	/// ref to screen object that displays in the middle of the UI
 	var/atom/movable/screen/map_view/ui_view
 
+	/// Theme of the mech TGUI
+	var/ui_theme = "ntos"
+	/// Module selected by default when mech UI is opened
+	var/ui_selected_module_index
+
 /datum/armor/sealed_mecha
 	melee = 20
 	bullet = 10
 	bomb = 10
 	fire = 100
 	acid = 100
 
-/obj/item/radio/mech //this has to go somewhere
-	subspace_transmission = TRUE
+/obj/vehicle/sealed/mecha/New(loc, built_manually, ...)
+	src.built_manually = built_manually
+	return ..()
```

Comment:
Can this not also be handled in Initialize?
It seems like if it was, you wouldn't even need to hold a reference to the variable as it is also only used in `Initialize`.